ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/main/resources/CH/VR-CH-0006/rule.json

```diff
@@ -17,7 +18,25 @@
   "Logic": {
     "if": [
       {
-        "var": "payload.v.0"
+        "and": [
+          {
+            "var": "payload.v.0"
+          },
+          {
```

Comment:
{
						"and": [
							{
								"var": "payload.v.0"
							},
							{
								"in": [
									{
										"var": "payload.v.0.mp"
									},
									[
										"EU/1/20/1528",
										"EU/1/20/1507",
										"EU/1/21/1529"
									]
								]
							}
						]
					},